src/style.css

```diff
@@ -0,0 +1,120 @@
+:root {
+  --secondary-color: #616070;
+  --buy-link-color: #00acdc;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.card__text {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__add-info,
+.card__rating-feedbacks {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0;
+}
+
+.card__add-info {
+  color: var(--secondary-color);
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  display: flex;
+  align-items: center;
+  /* align-content: center; */
+
+  justify-content: space-between;
+  height: 16px;
```

Comment:
looks like redundant height and it is bad practise to set height directly if it is not necessary